linux/shim/init.go

```diff
@@ -70,11 +73,14 @@ func newInitProcess(context context.Context, path string, r *shimapi.CreateReque
 		IO:            io,
 		NoPivot:       r.NoPivot,
 	}
+	sigchld := make(chan os.Signal, 2048)
+	// main.go calls Notify as well, but it does not hurt
+	signal.Notify(sigchld, syscall.SIGCHLD)
```

Comment:
you cannot have another sigchild handler here because it will just race with the main one